packages/grid/x-data-grid/src/tests/columns.spec.tsx

```diff
@@ -40,6 +41,29 @@ const RenderCellParamsExplicitTyping = () => {
             return params.formattedValue.toUpperCase();
           },
         },
+        {
+          field: 'price6',
+          type: 'actions',
+          getActions: (params: GridRowParams) => {
+            return params.row.foo.bar.toExponential();
+          },
+        },
```

Comment:
I don't think we need this test. The `row` in `GridRowParams` already has `any` so it accepts everything. Instead, you could create a different function using `GridColumns<{ ... }>`, then check if `getActions` receives the same row. Something like:


```ts
const columns: GridColumns<{ price: number }> = [{
  getActions: (params) => {
    return params.row.price.toUpperCase(); // fails
  }
}];
```